.github/workflows/release.yml

```
@@ -0,0 +1,80 @@
+name: Release
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      # update this after complete
+      - chore/test_release_workflow_123
```

避免真的觸發，先指定一個不存在的 branch

---

這邊可以先寫上去但註解起來，這樣比較知道最後其實會用什麼 
例如

```suggestion
      # - master
```

---

應該可以直接放 master 就好？
```suggestion
      - master
```

文件看起來是要 merge 進 master 的 PR 才會觸發，應該符合這邊的需求
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-based-on-the-head-or-base-branch-of-a-pull-request

---

對喔，如果不是 develop 可以直接寫